src/access/acp.test.ts

```diff
@@ -118,6 +125,7 @@ describe("hasInaccessiblePolicies", () => {
     );
     expect(internal_hasInaccessiblePolicies(resourceWithAcr)).toEqual(false);
   });
+
```

Comment:
Haha, I didn't want to "complain" about this missing line before, but I did notice it :P